code/docker-compose.yaml

```diff
@@ -12,3 +12,8 @@ services:
       - "9001:9001"
     volumes:
       - ./APIKEY.txt:/opt/etherpad-lite/APIKEY.txt
+  mail:
+    image: bytemark/smtp
+    restart: always
+    ports:
```

Comment:
Map the container port 25 (the classic port for SMTP) to 127.0.0.1:2525 used in our `application.yml`